python-main.js

```diff
@@ -481,6 +481,18 @@ function web_editor(config) {
     // the user sees their code or, in the case of a new program, uses some
     // sane defaults.
     function setupEditor(message, migration) {
+    
+     // Set the current language to bold in the dropdown menu
```

Comment:
The `setupEditor` function is to configure the ACE text editor wrapper (everything that starts with `EDITOR.xxx`). Maybe `setLanguage` is a better place for this?